packages/components/src/VirtualizedList/HeaderCheckbox/HeaderCheckbox.scss

```diff
@@ -1,6 +1,7 @@
 .tc-list-checkbox {
 	:global(.checkbox) {
 		margin: 0;
+		height: 12px;
```

Comment:
Would if to better to use $font-size-small to reflect the height we want to apply ?